src/handlebars/partials/menu.handlebars

```diff
@@ -8,7 +8,11 @@
       <li><a href="./releases.html">Latest release</a></li>
       <li><a href="./archive.html">Build archive</a></li>
       <li><a href="./nightly.html">Nightly builds</a></li>
-      <li><a href="./installation.html">Installation</a></li>
+      <li class="submenu"><a>Installation</a>
+        <ul>
+          <li><a href="./installers.html">Installers</a></li>
+          <li><a href="./installation.html">Archive files</a></li>
+        </ul>
```

Comment:
need to close li

Reply:
Structure changed - these comments no longer relevant. Resolving.